_docs_integrate/integration-example.md

```diff
@@ -191,7 +191,7 @@ For this, execute the `GET /api/v2/RelationshipTemplates/{id}` route (Accept Hea
 
 {% include rapidoc api_route_regex="^get /api/v2/RelationshipTemplates/{id}$" %}
 
-### App: Send a Relationship Request
+### App: Send a Pending Relationship
```

Comment:
I think we have to re-think our wording. This is the best example that just replacing all `Relationship Request` mentions with `Pending Relationship` is not correct.

Maybe we should talk about `creating a Relationship`? That this created Relationship always has the status Pending is a Detail imo.

@britsta & @Milena-Czierlinski wdym?

Reply:
"Creating a Relationship" would be correct in the sense that a data object of the type `Relationship` is created at this point in time. Nevertheless, I think that "creating a Relationship" in normal usage can be understood as creating a Relationship in status `"Active"`. I would therefore be in favor of adding that the Relationship is `"Pending"`.

Reply:
or we make sure that you can never "create" an active Relationship and you need a process for that.